domain/src/main/java/com/architect/codes/mu8/events/Event.kt

```diff
@@ -0,0 +1,7 @@
+package com.architect.codes.mu8.events
+
+data class Event(
```

Comment:
I think this data class fits in one line.

Reply:
Done